README.md

```diff
@@ -496,57 +385,63 @@ mvn clean verify -Dcode-quality
 
 Then open target/site/jacoco/index.html in your browser to see code coverage.
 
-If you want to see more of the code quality outputs, you must run the full site 
-generation from [above](code-quality)
+### Logging
 
-#### Install
+Logging is handled by [logback](Logback Home).  By default, the file logback.xml in the \<configDir\> will be 
+used.  You can point to another file with the *--logbackConfig* argument.  Feel free to modify the file
+in target/config if you want different logging.  The logback.xml is configured to only log to the console 
+currently.
 
-The *install* task runs after *verify* and copies any included
```

Comment:
This is Maven info, not Emissary info. We have already referenced the user to other documents that better explain this